source/sessions/driver-sessions.rst

```diff
@@ -553,17 +527,8 @@ to the server:
   the server at the other end of this connection supports sessions. This scenario
   will only be a problem until the next heartbeat against that server.
```

Comment:
"This scenario will only be a problem until the next heartbeat against that server." is no longer true and can be removed now.

Reply:
Updated the wording, thanks for the catch!